### PR TITLE
feat: multi selection

### PR DESCRIPTION
**Pull request by gitsad**
* Added multiselection functionality also with searchable enabled
* Added Agata in contributors
* Added tests for multiselection

### CODE REVIEW

**Review by koko57 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -61,6 +61,10 @@
       "name": "Marcin Sadowski",
       "url": "https://github.com/gitsad"
     },
+    {
+      "name": "Marcin Sadowski",
```
Comment (irekrog) [inline package.json:65]: name ≠ url 😄

File: package.json [hunk 2]
```diff
@@ -77,6 +81,7 @@
   },
   "dependencies": {
     "@gorhom/portal": "1.0.11",
+    "@gorhom/showcase-template": "^2.1.0",
```
Comment (irekrog) [inline package.json:84]: please delete it because it is needed only in example

File: src/components/multi-select/index.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                    option={null}
+                    optionWidth={'100%'}
+                    placeholderText={placeholderText}
+                    selectControlTextStyle={selectControlTextStyle}
+                />
+            );
+        }
+
+        const optionWidth = () => {
+            const WIDTH_TRESSHOLD = 100;
```
Comment (irekrog) [inline src/components/multi-select/index.tsx:75]: typo, should be `THRESHOLD`

File: src/components/multi-select/index.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+            let calculatedWidth = 100;
+            if (typeof initialWidth === 'number') {
+                calculatedWidth = (initialWidth - WIDTH_OFFSET) / length;
+                if (calculatedWidth < WIDTH_TRESSHOLD) {
+                    return WIDTH_TRESSHOLD;
+                }
+                return Math.floor(calculatedWidth);
+            }
+            if (typeof initialWidth === 'string') {
+                const screenWidth = Dimensions.get('window').width;
```
Comment (irekrog) [inline src/components/multi-select/index.tsx:88]: https://reactnative.dev/docs/usewindowdimensions is preffered

File: src/types/index.ts [hunk 5]
```diff
@@ -166,6 +166,13 @@ export interface SelectProps {
      *  @category Common
      */
     flatListProps?: Omit<FlatListProps<OptionType>, 'data' | 'renderItem' | 'ListEmptyComponent'>;
+    /**
+     *  if `true` then multi option can be picked
+     *
+     *  @category Common
+     *  @default false
+     */
+    multiSelection?: boolean;
```
Comment (irekrog) [inline src/types/index.ts:175]: also add this prop to `website/docs/api/common.md`

File: package.json (hunk 1 above)
Comment (gitsad) [reply package.json:65]: ou

File: src/components/multi-select/index.tsx (hunk 3 above)
Comment (gitsad) [reply src/components/multi-select/index.tsx:75]: 👍

File: src/components/multi-select/index.tsx (hunk 4 above)
Comment (gitsad) [reply src/components/multi-select/index.tsx:88]: 👍

File: src/types/index.ts (hunk 5 above)
Comment (gitsad) [reply src/types/index.ts:175]: 👍

**Review by irekrog — APPROVED** [review]

**Pull request merged**